Client/Frontend/Widgets/ActionOverlayTableViewController.swift

```diff
@@ -183,9 +202,16 @@ class ActionOverlayTableViewHeader: UITableViewHeaderFooterView {
     }
 
     func configureWithSite(site: Site, image: UIImage?, imageBackgroundColor: UIColor?) {
-        self.siteImageView.backgroundColor = imageBackgroundColor
-        self.siteImageView.image = image?.createScaled(SimpleHighlightCellUX.IconSize) ?? SimpleHighlightCellUX.PlaceholderImage
-        self.titleLabel.text = site.title.characters.count <= 1 ? site.url : site.title
-        self.descriptionLabel.text = site.tileURL.baseDomain
+        siteImageView.backgroundColor = imageBackgroundColor
+
+        if AppConstants.MOZ_AS_PANEL {
```

Comment:
I think what you said about the old TopSitePanel going away is correct. No need to spend time on something that's just going to go away. But I'd ask someone else for a second opinion just in case.